extensions/cloud/portability-cloud-microsoft/src/main/java/org/dataportabilityproject/cloud/microsoft/cosmos/MicrosoftCloudConstants.java

```diff
@@ -0,0 +1,13 @@
+package org.dataportabilityproject.cloud.microsoft.cosmos;
+
+/**
+ * Constants used by the Microsoft cloud extension.
+ */
+public interface MicrosoftCloudConstants {
+
```

Comment:
should these members be static final?

Reply:
I think they should be OK as is since interface members are by default static and final